### PR TITLE
Itk 5.4.0, Python CI

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-rleimage"
+version = "1.0.1"
+description = "ITKRLEImage provides run-length encoded storage for ITK images"
+readme = "README.rst"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Paul A. Yushkevich", email = "pauly2@mail.med.upenn.edu" },
```
Comment (dzenanz) [inline pyproject.toml:12]: Me and/or you should be added here too. Paul funded it and is using it in ITK-SNAP, but I doubt he would be able to help much with development questions, see https://github.com/KitwareMedical/ITKRLEImage/graphs/contributors.

Comment (thewtex) [reply pyproject.toml:12]: Paul wrote the original code, and he should have been attributed as commit author when it was imported from ITK-SNAP. I added you and I as authors though, too.

Comment (dzenanz) [reply pyproject.toml:12]: I mostly wrote the original code. After some performance-checking experimentation with `LabelMap`s, `RLEImage` approach was kicked off by this commit: https://github.com/pyushkevich/itksnap/commit/df6f0d15cdec0769c6fd7e4c0284538ce6dcdcb7. We later extracted `RLEImage`-related code into a remote module.

**Comment by dzenanz** [comment]
Windows packages fail with `Cannot find path 'D:\a\im\dist' because it does not exist.`

**Pull request merged**